DCA632FFFE5AEA98.json

```diff
@@ -1,24 +1,18 @@
 {
   "gateway_conf": {
-    "gateway_ID": "68C63AFFFF9FA90D",
+    "gateway_ID": "DCA632FFFE5AEA98",
     "servers": [
       {
         "server_address": "router.eu.thethings.network",
```

Comment:
Don't you want to add this one to v3? if so, you need to change this line to point to `eu1.cloud.thethings.network`

Reply:
68C63AFFFF9FA90D is a single server based on esp8266. I will reuse it as a node instead. The change of ID was suggested by git :-D    DCA6 ... is a RAK2245, so all channels are covered. Thank you for asking. Cheers

Reply:
That's not really the question ;)  The thing is that the server URL you used (`router.eu.thethings.network`) is V2, but you want to connect the gateway to V3 I guess. In any case, I will merge, and if you want to migrate this to V3 later on, just update the address to `eu1.cloud.thethings.network`